src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebDocumentRouteConfig.kt

```diff
@@ -7,12 +7,18 @@ import org.vaccineimpact.orderlyweb.controllers.web.DocumentController
 object WebDocumentRouteConfig : RouteConfig
 {
     private val readDocuments = setOf("*/documents.read")
+    private val manageDocuments = setOf("*/documents.manage")
     override val endpoints: List<EndpointDefinition> = listOf(
             WebEndpoint("/project-docs/*",
                     DocumentController::class, "getDocument", contentType = ContentTypes.binarydata)
                     .secure(readDocuments),
             WebEndpoint("/project-docs/",
                     DocumentController::class, "getAll")
-                    .secure(readDocuments))
+                    .secure(readDocuments),
+            WebEndpoint("/documents/refresh/", DocumentController::class, "refreshDocuments")
```

Comment:
Have to keep this prefix as "/documents/" rather than "project-docs" to avoid route ambiguity